modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -69,8 +69,7 @@ spec:
       podConfig:
         nodeSelector: <node_selector> <4>
   backupLocations:
-    - name: default
-      velero:
+    - velero:
```

Comment:
name is not supported for  snapshotLocations but it if for backupLocations
please undo this change